openstack/cinder/templates/type-seeds.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Release.IsUpgrade }}
+{{- if .Capabilities.APIVersions.Has "openstack.stable.sap.cc/v1"}}
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: cinder-type-seed
+  labels:
+    component: cinder
+    app: cinder
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    type: seed
```

Comment:
I'm not sure about these. the `cinder-seed` only has a `name` as metadata. They probably will not matter.

Reply:
removed